benchmark/compare.js

```diff
@@ -1,10 +1,12 @@
 var usage = 'node benchmark/compare.js ' +
             '<node-binary1> <node-binary2> ' +
-            '[--html] [--red|-r] [--green|-g]';
+            '[--html] [--red|-r] [--green|-g] ' +
+            '[-- <type> [testFilter]]';
 
 var show = 'both';
 var nodes = [];
 var html = false;
+var benchmarks;
```

Comment:
I'm not sure what you mean. AFAIK using `require('./common').createBenchmark()` won't work well because it spawns the test with `stdio: 'inherit'`, so you can't capture the output needed for comparing.